src/wallet/sqlite.cpp

```diff
@@ -135,6 +132,9 @@ SQLiteDatabase::SQLiteDatabase(const fs::path& dir_path, const fs::path& file_pa
         if (ret != SQLITE_OK) {
             throw std::runtime_error(strprintf("SQLiteDatabase: Failed to initialize SQLite: %s\n", sqlite3_errstr(ret)));
         }
+        if (g_sqlite_count == 1) {
+            LogInfo("Using SQLite Version %s", SQLiteDatabaseVersion());
+        }
```

Comment:
what about (and no longer need to check `g_sqlite_count`)...
```suggestion
        static bool sqlite_version_logged = false;
        if (!sqlite_version_logged) {
            LogInfo("Using SQLite Version %s", SQLiteDatabaseVersion());
            sqlite_version_logged = true;
        }
```